### PR TITLE
feat: replaced smartdocuments cert with their wildcard cert

### PR DESCRIPTION
**Pull request by infojohn**
valid until march 2025

Solves: PZ-1582

### CODE REVIEW

**Review by RickWoltheus — APPROVED** [review]

File: certificates/smartdocuments/-text
[the remark this answers is not in the document]
Comment (infojohn) [reply certificates/smartdocuments/-text:1]: I agree, and done!

File: certificates/smartdocuments/text
[the remark this answers is not in the document]
Comment (infojohn) [reply certificates/smartdocuments/text:1]: I agree, and done!

**Review by hsiliev — APPROVED** [review]

**Review by bas-info-nl — APPROVED** [review]

**Comment by edgarvonk** [comment]
Nice! You tested this locally right? Good thing to do because this is not covered by any automated tests yet.

**Review by edgarvonk — APPROVED** [review]
Looks good but see comment. :-)

**Comment by hsiliev** [comment]
Verified via https://github.com/infonl/dimpact-zaakafhandelcomponent/pull/636

**Pull request merged**